lib/Room.js

```diff
@@ -24,7 +24,7 @@ class Room extends EventEmitter {
    *   mediasoup Router must be created.
    * @param {String} roomId - Id of the Room instance.
    */
-  static async create({ mediasoupWorker, roomId, consumerReplicas }) {
+  static async create({ mediasoupWorker, roomId, consumerReplicas, maxPeersRoom }) {
```

Comment:
```suggestion
  static async create({ mediasoupWorker, roomId, consumerReplicas, maxPeerCount }) {
```

Reply:
Fixed all definition change maxPeersRoom to maxPeersCount